torch/csrc/jit/serialization/flatbuffer_serializer.cpp

```diff
@@ -364,10 +390,18 @@ flatbuffers::DetachedBuffer FlatbufferSerializer::serializeModule(
   for (const auto& ival : jit_constants) {
     jit_constants_indexes.emplace_back(storeIValueAndGetIndex(fbb, ival));
   }
+  const uint32_t version =
+      kMinVersion > caffe2::serialize::kProducedBytecodeVersion
+      ? kMinVersion
+      : caffe2::serialize::kProducedBytecodeVersion;
+  const uint32_t operator_version =
+      static_cast<uint32_t>(module.min_operator_version());
+  const uint32_t bytecode_version =
+      static_cast<uint32_t>(module.bytecode_version());
 
   auto mod = CreateModule(
       fbb,
-      0, /* version */
+      version,
```

Comment:
Seems not needed.
```suggestion
```

Reply:
kProducedBytecodeVersion is 8 currently. Changing it to 9 now will also change the pickle files generated.